cache/lru/lru_test.go

```diff
@@ -0,0 +1,67 @@
+package lru
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCreate(t *testing.T) {
+
+	c, err := Create(-1)
+	assert.Nil(t, c)
+	assert.Error(t, err)
+
+	c, err = Create(512)
+	assert.NotNil(t, c)
+	assert.NoError(t, err)
+}
+
+func TestCacheOperations(t *testing.T) {
```

Comment:
Let's use sub-tests for the cases below, makes it more readable

Reply:
Done, resolving this comment :)